### ISSUE TITLE
Reducing coring times

### ISSUE THREAD

**Issue by Firesoul7** [PhelanBavaria/ancienttimeline#29]
Coring times need to be greatly reduced, such that they are near zero (unless this has already been done). I also think that making cores should be a little cheaper. Not much, just a little.

**Comment by PhelanBavaria** [comment]
I thought about that yesterday, and I agree. Additionally to that overextension should be reduced dramatically and how long cores stick around should be increased by quite a bit.

**Comment by qweytr** [comment]
I already suggested increasing the administrative efficiency. That reduces overextension and coring costs. I'm not sure how that affects the coring times though.

**Comment by PhelanBavaria** [comment]
Are those the only two things that administrative efficiency influences?

**Comment by qweytr** [comment]
Also war score costs, but that's all as far as I know.

**Comment by PhelanBavaria** [comment]
I think reducing war score cost should be a thing too, so probably our suggestion would be the best.

**Comment by Firesoul7** [comment]
When will this be done?

**Comment by PhelanBavaria** [comment]
Rethinking this, I don't think that the core creation time should be 0.
qweytr, could you do this by adding a modifier? Maybe this could be one part of the centralization system?

**Comment by qweytr** [comment]
Can coring times even be decreased by modifiers? I only know of coring cost modifiers.

**Comment by PhelanBavaria** [comment]
I meant the administrative efficiency.

**Comment by qweytr** [comment]
That doesn't decrease coring time as far as I know, only coring cost, but I might be wrong.

**Comment by PhelanBavaria** [comment]
Oh, I see, I guess we can only decrease coring time in the defines.lua.
Perhaps we just leave that be for now and see how it plays.

**Comment by Firesoul7** [comment]
Is there any problem with editing the defines.lua?

**Comment by qweytr** [comment]
Not really, but there are other areas of the world where reduced coring times wouldn't work that well. Reduced coring times would even make expansion that much easier, as too high overextension is quite bad even for a short time. Reduced overextension is a better bonus and that is provided by the increased administrative efficiency.
